### PR TITLE
fix(pie-monorepo): DSW-1969 updated Playwright and Percy dependencies TEST

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
## Describe your changes (can list changeset entries if preferable)
- Updated Percy and Playwright dependencies
- Updated tests to support updated Playwright deps

A couple things to note with this PR is that all screenshots added are now covering the Chrome browser.
Some differences in the screenshots of existing packages are due to the change of order in some of the listed variations.

Another note is there is 1 test that has been skipped. This seems to be a legitimate failure of the component test that is blocking this work so a ticket will be written to address this.

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving

### Reviewer 1 - @xander-marjoram 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: packages/components/pie-modal/test/component/pie-modal.spec.ts [hunk 1]
```diff
@@ -417,14 +419,17 @@ test.describe('modal', () => {
                 expect(isModalVisible).toBe(true);
             });
 
-            test('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
+            test.skip('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
```
Comment (raoufswe) [inline packages/components/pie-modal/test/component/pie-modal.spec.ts:422]: expected to skip?

Comment (JoshuaNg2332) [reply packages/components/pie-modal/test/component/pie-modal.spec.ts:422]: Oh yes, sorry, forgot to add a comment in the PR for this. Will do that now.

This is a legitimate failure that needs to be looked at but was blocking my PR. Will write a ticket to address this issue.

Comment (JoshuaNg2332) [reply packages/components/pie-modal/test/component/pie-modal.spec.ts:422]: Update: DSW-2218 has been created to address this test.

File: .github/workflows/amplify-deploy.yml [hunk 2]
```diff
@@ -106,7 +106,7 @@ jobs:
           aws amplify start-deployment \
             --app-id ${{ env.AMPLIFY_ID }} \
             --branch-name ${{ env.BRANCH_NAME }} \
-            --source-url https://${{ env.BUCKET_NAME }}.s3.${{ env.AWS_REGION }}.amazonaws.com/${{ env.ZIP_NAME }}
+            --source-url s3://${{ env.BUCKET_NAME }}/${{ env.ZIP_NAME }}
```
Comment (xander-marjoram) [inline .github/workflows/amplify-deploy.yml:109]: This is much cleaner! Does this fix any particular issues? And do you think this was always possible?

Comment (JoshuaNg2332) [reply .github/workflows/amplify-deploy.yml:109]: I think this was always possible but the problem it solves was the deployments failing on Pie Docs. We'll probably look to change this in other places using this same deployment format.

**Review by leksaBoiko — DISMISSED** [review]

File: .changeset/fluffy-numbers-deny.md
Comment (xander-marjoram) [inline .changeset/fluffy-numbers-deny.md:1]: I don't think this file should be showing, it must have snuck in during a rebase

**Pull request closed without merging**